Gemfile.lock

```diff
@@ -395,4 +408,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.10.6
+   1.13.1
```

Comment:
This might cause us a problem. See: https://github.com/alphagov/publishing-api/pull/536#pullrequestreview-2506357 we have 1.11.2 in govuk_puppet.